src/components/header/index.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from "prop-types"
 import React from "react"
 
 const Header = ({ siteTitle }) => (
-  <header
+  <div
```

Comment:
See my comment about the Helmet – is it possible to revert these changes, and focus testing on the components inside page 2 only?